cmd2.py

```diff
@@ -1922,6 +1946,92 @@ def _cmdloop(self):
 
             return stop
 
+    @with_argument_list
+    def do_alias(self, arglist):
```

Comment:
This looks pretty clean and similar to Bash, so it should probably be self-explanatory to many people.